src/ResourceManager/Storage/Commands.Management.Storage/StorageAccount/GetAzureStorageAccountLastSyncTime.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Management.Storage.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Storage;
+using Microsoft.Azure.Management.Storage.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Management.Storage
+{
+    [Cmdlet("Get", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + StorageAccountLastSyncTimeNounStr, DefaultParameterSetName = AccountNameParameterSet), OutputType(typeof(PSManagementPolicy))]
+    public class GetAzureStorageAccountLastSyncTimeCommand : StorageAccountBaseCmdlet
+    {
+        /// <summary>
+        /// AccountName Parameter Set
+        /// </summary>
+        private const string AccountNameParameterSet = "AccountName";
```

Comment:
Should also have a resourceId parameter set - this can be added when it moves back into preview

Reply:
See , thanks！
Will keep in mind when move it back to preview.